src/views/asset-view/asset-view.test.tsx

```diff
@@ -203,4 +205,50 @@ test("renders the asset view for invalid asset type", async () => {
   await screen.findByText(locale.assetCouldNotBeLoaded);
 });
 
-//extra tests for retrieving personId to be added here once tested that it is working
+test("renders the asset view for missing person id", async () => {
+  mockActiveTenureV1.householdMembers[0].id = "2d9d6ac5-d376-4ac4-9a00-85659be82d10";
+  mockActiveTenureV1.householdMembers[0].fullName = "FAKE_Alice FAKE_Rowe";
+
+  const alertsResponse = {
+    propertyReference: mockAssetV1.assetId,
+    uprn: null,
+    addressNumber: null,
+    alerts: [
+      {
+        dateModified: "12/02/2022",
+        modifiedBy: "GoogleSheet",
+        startDate: "12/02/2022",
+        endDate: null,
+        alertCode: null,
+        description: "Domestic Abuse Case - please seek advice",
+        reason: null,
+        assureReference: "123456",
+        personName: "FAKE_Alice FAKE_Rowe",
+        personId: null,
+      },
+    ],
+  };
+
+  server.use(
+    rest.get("/api/v1/assets/:id", (req, res, ctx) =>
+      res(ctx.status(200), ctx.set("ETag", '"2"'), ctx.json(mockAssetV1)),
+    ),
+    rest.get(`/api/v1/cautionary-alerts/properties-new/:propertyId`, (req, res, ctx) =>
+      res(ctx.status(200), ctx.json(alertsResponse)),
+    ),
+    rest.get("/api/v1/tenures/:tenureId", (req, res, ctx) =>
+      res(ctx.status(200), ctx.set("ETag", '"2"'), ctx.json(mockActiveTenureV1)),
+    ),
+  );
+
+  render(<AssetLayout assetDetails={mockAssetV1} />, {
```

Comment:
This works with AssetView too, may as well use the same as the other tests if so